OCAExplorer/src/App.tsx

```diff
@@ -17,7 +17,7 @@ import { StyledEngineProvider, ThemeProvider } from "@mui/material/styles";
 const setNewUserCookie = () => {
     document.cookie = 'OCAExplorerSeenDemo=true; path=/;';
 }
-const seenDemo: string | undefined = document.cookie.split('; ').find((c) => c.split("=")[0] == 'OCAExplorerSeenDemo');
+const seenDemo: () => string | undefined = () => document.cookie.split('; ').find((c) => c.split("=")[0] == 'OCAExplorerSeenDemo');

```

Comment:
Thanks for the tip! I'll shift the logic over to useMemo rather than repeated queries. I'll also see if I can have an alternative for disabled cookies